go.mod

```diff
@@ -4,23 +4,27 @@ go 1.14
 
 require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.0
-	github.com/go-errors/errors v1.0.1
+	github.com/go-errors/errors v1.4.0
```

Comment:
This updates a lot of dependencies. Do we need to update all of this for kyaml? In particular, I'm not sure we need to update the cli-utils library here.

Reply:
same as above comment, we need to update cli-utils or we get a build error